app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -347,7 +347,7 @@ export default {
           await this.$store.dispatch('sendMessage', messagePayload);
           this.$emit('scrollToMessage');
         } catch (error) {
-          // Error
+          this.showAlert(this.$t('CONVERSATION.FILE_FORMAT_ERROR'));
```

Comment:
```suggestion
          const errorMessage = error?.response?.message || this.$t('CONVERSATION.MESSAGE_ERROR');
          this.showAlert(errorMessage);
```

Add a generic value for `MESSAGE_ERROR`